src/plugins/intel_gpu/src/graph/network.cpp

```diff
@@ -1246,21 +1265,41 @@ void network::execute_impl(const std::vector<event::ptr>& events) {
 
         execute_primitive(inst, events);
 
+        // Dump output buffers of 'inst'
         GPU_DEBUG_IF(debug_config->dump_layers_path.length() > 0) {
             get_stream().finish();
             const std::string layer_name = inst->id();
             auto prog_id = ((get_program() != nullptr) ? get_program()->get_id() : 0);
             auto net_id = get_id();
             GPU_DEBUG_IF(debug_config->is_target_iteration(curr_iter) &&
-                        debug_config->is_dumped_layer(layer_name, inst->is_output())) {
-                for (size_t i = 0; i < get_primitive(inst->id())->outputs_memory_count(); i++) {
-                    log_memory_to_file(get_primitive(inst->id())->output_memory_ptr(i),
-                                    get_stream(),
-                                    "program" + std::to_string(prog_id) +
-                                    "_network" + std::to_string(net_id) +
-                                    "_" + get_iteration_prefix(curr_iter) +
-                                    layer_name + "_dst" + std::to_string(i),
-                                    debug_config->dump_layers_raw);
+                        debug_config->is_dumped_layer(layer_name, inst->is_output(), inst->is_input())) {
```

Comment:
* I think it conflicts with the behavior of "DumpPath" as DumpPath means enabling layer dump.
* Can we guess the layer name from the filename so that we can just list the filenames?

Reply:
I think it conflicts with the behavior of "DumpPath" as DumpPath means enabling layer dump.
> I thought it was fine with 'is_xxx'. The dumping configs start with dump_. Do you have any idea you want to change?
  
Can we guess the layer name from the filename so that we can just list the filenames?
> some layers' names contain '/' which is replaced by '_'. It causes additional effort for guessing layer name. While dumping binary file, debug msg showed input layer name. I think it would be enough helpful for just debugging.

Reply:
is_dumped_layer -> is_layer_for_dumping
is_loading_layer -> is_layer_fo_loading